File: yoroflow-ui/src/app/shared-module/yorogrid/headers-vo.ts
export class HeadersVO {
    checkboxEnabled: boolean;
    filterEnabled: boolean;
    exportEnabled: boolean;

    tableWidth: number;
    defaultPageSize: number;
    defaultSortColumn: string;

    headers: HeaderVO[];
}

export interface HeaderVO {
    headerId: string;
    headerName: string;

    sortable: boolean;
    filterable: boolean;
    hidden: boolean;
    enabled: boolean;
    parameterOnClick: boolean;

    widthPercentage: number;

    fieldDataType: string;
    columnSeqNo: number;
    style:any;
}

File: support-objects/src/main/java/com/yorosis/yoroapps/grid/vo/HeadersVO.java
package com.yorosis.yoroapps.grid.vo;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class HeadersVO {
	private List<HeaderVO> headers;
	private boolean checkboxEnabled;
	private boolean filterEnabled;
	private boolean exportEnabled;

	private int tableWidth;
	private int defaultPageSize;
	private String defaultSortColumn;

}